### ISSUE TITLE
Update work unit description to combine all issues

### ISSUE THREAD

**Issue by zanerock** [liquid-labs/liq-work#79]
## Overview

The work unit description becomes the PR title, which gets encoded in the commit message which we will eventually use to generate changelogs. So, when we have multiple issues, we should combine their titles in the work unit description.

## Deliverables

- [ ] Ensure combined title created in `work start` route.
- [ ] Ensure combined title created in `work issues add` route.
- [ ] Update title in `work issues remove` route.

Basically, we should regenerate the description anytime issues are modified.
